server-plugins/notification-resources/src/index.ts

```diff
@@ -862,23 +871,31 @@ export async function createCollabDocInfo (
         objectId: object._id,
         user: { $in: unsubscribe }
       })
-      await removeContexts(notifyContexts, unsubscribe, control)
+      await removeContexts(ctx, notifyContexts, unsubscribe, control)
     }
 
     return res
   }
 
-  const notifyContexts = await control.findAllCtx(ctx, notification.class.DocNotifyContext, { objectId: object._id })
-  await removeContexts(notifyContexts, unsubscribe, control)
-  await updateContextsTimestamp(notifyContexts, originTx.modifiedOn, control, originTx.modifiedBy)
+  let notifyContexts: DocNotifyContext[] = []
+  if (!(object._id === tx.objectId && tx._class === core.class.TxCreateDoc)) {
```

Comment:
Why we should skip this case?

Reply:
Because document is not yet created, it could not have DocNotifyContext yet

Reply:
got it!